apps/docs/components/components/badge.md

```diff
@@ -73,7 +73,7 @@ A common use case for the Badge is to place it on a user's avatar.
 
 | Prop name   | Type               | Default value | Possible values                                        |
 | ----------- | ------------------ | ------------- | ------------------------------------------------------ |
-| `content`   | `string            | number`       |                                                        |
+| `content`   | `string  | number`       |                                                        |
```

Comment:
Ah sorry im confused now, is it `string |  number` ? if so there is missing one `|` at the end. If not then string and number exclude themselves and have missing `

Reply:
@Szymon-dziewonski done